drizzlepac/devutils/search_skyframes.py

```diff
@@ -0,0 +1,316 @@
+#!/usr/bin/env python
+
+""" This script allows the user to query a master observations .csv file by any combination of exposure name,
+    skycell name, instrument/detector, and/or filter name. The results are printed the screen and optionally
+    written to a user-specified output .csv file. The results will be sorted first by skycell name, then by
+    instrument/detector, filter name and finally by image name."""
```

Comment:
Please include at least one simple example of how to use this code in the module level docstring here.   The example should start with how to point to the master file, how to define a search, how to perform the search, and how to visualize the results.  A clear as the code is to follow, it hides a lot of details in the calls it uses.  This will also help anyone trying to either maintain or just simply use this code in the future to know how this code was designed and intended to be used.

Reply:
Issue resolved by commit [44205e0](https://github.com/spacetelescope/drizzlepac/pull/964/commits/44205e0991622893e87586fea26e1381a5ee9d24)

Reply:
issue resolved by commit [d96c85b5](https://github.com/spacetelescope/drizzlepac/pull/964/commits/d96c85b53ac5931851995bfcfd89ede8e8e51e90)